![logo](https://raw.githubusercontent.com/mx-417/GTOP/main/20251226141013110.png)
# Genotype Tissues-Plasma Omics Project

GTOP is a long-read multi-omics atlas comprising transcriptomic and genetic variation data from 33 tissues of 160 healthy Chinese donors. The project expands tissue coverage beyond existing multi-tissue resources and enables systematic characterization of regulatory variants, complex genomic features, and transcript isoform diversity that are poorly captured by short-read sequencing. As a population-representative resource for East Asians, GTOP addresses a major gap in current functional genomics references and supports studies of regulatory variation, disease mechanisms, and population-specific genetic architecture.
## Data Access
---
### Raw reads
---
Newly generated sequencing data for the 160 individuals is available on the Sequence Read Archive (Accession: **[PRJNA851328](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA851328)**).

### Processed data
---
Processed gene expression matrices and QTL mapping results (as well as a host of other downstream data) are currently available on Zenodo (**[MAGEv1.0 Zenodo link](https://zenodo.org/doi/10.5281/zenodo.10535719)**) as well as Dropbox (**[MAGEv1.0 Dropbox link](https://www.dropbox.com/scl/fo/3filfi60qhxgokmjb6weq/AE14SzbtRK1U2mfstLM8jFc?rlkey=ihnv8hpbj1ilb96cxsm8fc42p&dl=0)**).

Briefly, this repo contains the following data:

1. Sample metadata and sequencing metrics
2. Gene expression and splicing matrices used for e/sQTL mapping and analyses of global trends of expression/splicing diversity
3. cis-e/sQTL mapping results, including aFC estimates for cis-eQTLs
4. Functional annotations of cis-e/sQTLs
5. Results of colocalization analysis between MAGE e/sQTLs and complex trait GWAS from the [PAGE](https://doi.org/10.1038/s41586-019-1310-4) study
6. Results of analyses of global trends of expression/splicing diversity
7. Jointly-generated top genotype PCs for samples in GTOP and other resources with paired WGS/RNA-seq data (Geuvadis, GTEx, AFGR)

READMEs are provided for all data in the repo.

If you are having trouble accessing these data, please feel free to contact us to explore other options.

### Variant calls
---
The high-coverage variant calls used for QTL mapping were previously generated by the [New York Genome Center (NYGC)](https://doi.org/10.1016/j.cell.2022.08.004) and are available through the 1KGP [FTP site](http://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000G_2504_high_coverage/working/20201028_3202_phased/).

## Code
---
Code used for data processing and downstream analyses is made available in the [analysis_pipeline/](https://github.com/mccoy-lab/MAGE/blob/main/analysis_pipeline) directory, along with READMEs describing how each script is run.

Code used to produce major figures/panels in the manuscript is made available in the [figure_generation/](https://github.com/mccoy-lab/MAGE/blob/main/figure_generation) directory.

## The MAGE manuscript
---
For more information about the MAGE resource as well as analyses performed using this resource, please see our paper:

**Long-read multi-omics reveals the regulatory architecture of East Asian populations across 33 human tissues**  

### Citing GTOP
---
If you use GTOP data in your own work, please cite the paper linked above.